This is ci error log, please take a look:
````
CMake Error at CMakeLists.txt:12 (target_link_libraries):
  Target "openfbx" links to:

    libdeflate::libdeflate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
````

--- a/ports/openfbx/CMakeLists.txt
+++ b/ports/openfbx/CMakeLists.txt
@@ -9,7 +9,8 @@ include(GNUInstallDirs)
 find_package(libdeflate REQUIRED)
 
 add_library(openfbx src/ofbx.cpp)
-target_link_libraries(openfbx PRIVATE libdeflate::libdeflate)
+target_link_libraries(openfbx PRIVATE $<IF:$<TARGET_EXISTS:libdeflate::libdeflate_shared>,libdeflate::libdeflate_shared,libdeflate::libdeflate_static>)
+
 
 target_include_directories(openfbx
         PUBLIC

--- a/versions/o-/openfbx.json
+++ b/versions/o-/openfbx.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "0044aa1e6ff32e6287babe258cab76c7f7bf8bf7",
+      "git-tree": "91f58df63150443bd8e644f319361162031b88c7",
       "version-date": "2024-05-08",
       "port-version": 1
     },

